docs/language.rst

```diff
@@ -351,6 +351,17 @@ Values can be assigned to the remaining properties as if they are variables. To
     // adding a value twice does not affect the relation,
     // s1.files still equals [f1, f2, f3]
 
+In addition, attributes can be assigned in a constructor using keyword arguments by using `**dct` where `dct` is a dictionary that contains
+attribute names as keys and the desired values as values. This method does however have a small performance impact. For example:
+
+.. code-block:: inmanta
+
+    Host.files [0:] -- File.host [1]
+    h1 = Host("test")
+
+    file1_config = {"path": "/opt/1"}
+    f1 = File(host=h1, **file1_config)
```

Comment:
This line results in:
![langref](https://user-images.githubusercontent.com/7672159/72554562-3d6f1d80-389b-11ea-9310-c53b61a4a7f5.png)

I assume I need to add ** as a supported operator to the language for sphinx? How do I do this?

Reply:
https://github.com/inmanta/inmanta-sphinx/blob/master/sphinxcontrib/inmanta/pygments_lexer.py

Reply:
thanks